include/r502_types.h

```diff
@@ -69,7 +69,9 @@ typedef enum CommandType {
 
     WriteNotepad,
 
-    ReadNotepad
+    ReadNotepad,
+
+    AuraLedConfig
```

Comment:
Please provide a short description of what the command does

Reply:
Done!